runbotics/runbotics-orchestrator-ui/src/views/process/ProcessMainView/ProcessMainView.tsx

```diff
@@ -37,7 +64,7 @@ const ProcessMainView: FC = () => {
                         value={tab}
                         variant="scrollable"
                     >
-                        {processTabs.map((processTab) => (
+                        {accessedTabs.length > 1 && accessedTabs.map((processTab) => (
```

Comment:
`accessedTabs.length` dont need `> 1` if it 0 it is translated into false everything above is true

Reply:
the point is to display tabs when there's more than one tab: when ` process.Tabs.length >= 2`. Because, when there's only one tab, you can't switch to another, so there's no point to display any tabs.